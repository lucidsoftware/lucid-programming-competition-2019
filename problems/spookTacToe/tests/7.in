11 11
#.....#....
x#.....##.#
...........
#..x.......
.....#.....
....#......
##...#.....
x...#......
.o.o.......
..#........
x#.#.......