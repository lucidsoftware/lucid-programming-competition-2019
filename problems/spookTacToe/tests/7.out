#oxoxo#oxox
x#xoxox##o#
oxoxoxoxoxo
#xoxoxoxoxo
xoxox#xoxox
xoxo#oxoxox
##oxo#oxoxo
xxox#xoxoxo
ooxoxoxoxox
xo#oxoxoxox
x#x#oxoxoxo